Add episode 42 (#372)
trinodb/trino.io: _episodes/42.md

* Martin Traverso, Trino creator and CTO at [Starburst](https://starburst.io)
  ([@mtraverso](https://twitter.com/mtraverso))

## Releases 402 to 403

Official highlights from Martin Traverso:

[Trino 402]({{site.url}}/docs/current/release/release-402.html)

* Support for column comments in Hive and Iceberg views.
* Support predicate pushdown on temporal types in MongoDB connector.
* Faster `OR`, `nullif`, and arithmetic operations in SQL Server connector.

[Trino 403]({{site.url}}/docs/current/release/release-403.html)

* Support for `DELETE` in MongoDB.
* Faster aggregations.
* Faster data transfers with fault-tolerant execution.
* Faster `SHOW SCHEMAS` in BigQuery.
* Faster `expire_snapshots` in Apache Iceberg.

More detailed information is available in the release notes for
[Trino 402]({{site.url}}/docs/current/release/release-402.html),
and
[Trino 403]({{site.url}}/docs/current/release/release-403.html).

## Trino Summit 2022 recap

This episode we're doing a recap of both the Trino Summit and the first Trino
Contributor Congregation. We dive into what everyone's favorite Trino Summit
sessions were. Then we cover key takeaways from the Trino Contributor

> Congregation, which took place the day after and some of the topics we went over

Reviewer: after.

and get rid of the rest

  [Starburst](https://starburst.io)
* Manfred Moser, Director of Information Engineering at 
  [Starburst](https://starburst.io)
  ([@simpligility](https://twitter.com/simpligility))

## Guests

* Brian Zhan, Product Manager at [Starburst](https://starburst.io)
  ([@brianzhan1](https://twitter.com/brianzhan1))
* [Claudius Li](https://www.linkedin.com/in/claudiusli), Product Manager at
  [Starburst](https://starburst.io)
* Dain Sundstrom, Trino creator and CTO at [Starburst](https://starburst.io)
  ([@daindumb](https://twitter.com/daindumb))
* Martin Traverso, Trino creator and CTO at [Starburst](https://starburst.io)
  ([@mtraverso](https://twitter.com/mtraverso))

## Releases 402 to 403

Official highlights from Martin Traverso:

[Trino 402]({{site.url}}/docs/current/release/release-402.html)

[Trino 403]({{site.url}}/docs/current/release/release-403.html)

* Support for `DELETE` in MongoDB.
* Faster aggregations.
* Faster data transfers with fault-tolerant execution.
* Faster `SHOW SCHEMAS` in BigQuery.
* Faster `expire_snapshots` in Apache Iceberg.

Additional highlights worth a mention according to Cole:

> * 

Reviewer 2: I think we can skip this because we didn't talk about anything extra.

More detailed information is available in the release notes for
[Trino 402]({{site.url}}/docs/current/release/release-402.html),
and
[Trino 403]({{site.url}}/docs/current/release/release-403.html).

## Trino Summit 2022 recap

This episode we're doing a recap of both the Trino Summit and the first Trino

> Contributor Congregation. We dive into what everyone's favorite Trino Summit
> sessions are. Then we finally cover key takeaways from the Trino Contributor

Reviewer 2: ```suggestion
Contributor Congregation. We dive into what everyone's favorite Trino Summit
sessions were. Then we cover key takeaways from the Trino Contributor
```

> Congregation that took place the day after and some of the topics we went over

Reviewer 2: ```suggestion
Congregation, which took place the day after and some of the topics we went over
```
